pkg/k8shandler/rbac.go

```diff
@@ -94,6 +95,78 @@ func NewClusterRoleBinding(bindingName, roleName string, subjects []rbac.Subject
 	}
 }
 
+// CreateOrUpdateRole creates and returns a new role resource or updates an existing or returns an error
+func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateRole(name, namespace string, rules []rbac.PolicyRule) (*rbac.Role, error) {
+	role := NewRole(name, namespace, rules)
+	utils.AddOwnerRefToObject(role, utils.AsOwner(clusterRequest.cluster))
+
+	if err := clusterRequest.Create(role); err != nil {
+		if !errors.IsAlreadyExists(err) {
+			return nil, fmt.Errorf("Failure creating role for %q: %v", clusterRequest.cluster.Name, err)
+		}
+
+		err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+			return clusterRequest.Update(role)
+		})
+
+		if err != nil {
+			return nil, fmt.Errorf("Failure updating role for %q, %v", clusterRequest.cluster.Name, err)
+		}
+
+		return role, nil
+	}
+
+	return role, nil
+}
+
+// RemoveRole deletes a role or returns error
+func (clusterRequest *ClusterLoggingRequest) RemoveRole(name, namespace string) error {
+	role := NewRole(name, namespace, []rbac.PolicyRule{})
+
+	err := clusterRequest.Delete(role)
+	if err != nil && !errors.IsNotFound(err) {
+		return fmt.Errorf("Failure deleting role for %q: %v", clusterRequest.cluster.Name, err)
+	}
+
+	return nil
+}
+
+// CreateOrUpdateRoleBinding creates and returns a new rolebinding resource or updates an existing or returns an error
+func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateRoleBinding(name, namespace, roleName string, subjects []rbac.Subject) (*rbac.RoleBinding, error) {
```

Comment:
I dont think any of this is needed if we are relying on static RBAC genertion

Reply:
This methods are dropped because of zero usage in the code base.